woocommerce/woocommerce-ios
# Store creation MVP: add entry point to prologue after account creation

Part of #7891 
Part of #7879 

⚠️ Please only review this PR after the base PR https://github.com/woocommerce/woocommerce-ios/pull/7906 is approved



### Description


In https://github.com/woocommerce/woocommerce-ios/pull/7906, account creation can be initiated from the login prologue (tentatively from the `New to WooCommerce` CTA). This PR integrates account creation completion with store creation, by extracting the store creation webview logic from `StorePickerViewController` to a new `StoreCreationCoordinator`.

### Testing instructions


- Launch the app
- Log out if needed
- Tap on the `New to WooCommerce?` CTA on the prologue screen --> it should navigate to a form to create a WPCOM account as in the screenshot
- Enter a new email and password, then tap on the submit CTA --> after a bit, it should open the store creation view
- Proceed to finish store creation --> after a new Jetpack site is available, the view should be dismissed and the app should be connected to the new store

---
